settings.gradle.kts

```diff
@@ -10,6 +10,7 @@ pluginManagement {
 
 plugins {
   id("com.gradle.develocity") version("3.19")
+  id("com.android.settings") version "8.8.0" // Keep in sync with agp in libs.versions.toml
```

Comment:
Unfortunately, we cannot reference the version catalog from the settings file since we are in fact loading the version catalog here. It’s a bit of chicken and egg problem.

Reply:
FYI the android studio built-in AGP Upgrade Assistant automatically updates this for you when you run it. It keeps both this and the version catalog in sync